### PR TITLE
44 feature table of contents for readme

### PR DESCRIPTION
**Pull request by luke-welbourn**
ReadMe and button SCSS hopefully done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,144 +1,104 @@
+# Project Name
```
Comment (Charlie-robin) [inline README.md:1]: Can you change this?

File: README.md [hunk 2]
```diff
@@ -208,94 +181,134 @@ type RegisterProps = {
...
-{filteredSearch.map((event: MockEvent, index: number))}
-```
+- **Error Message**: When a user lands on a non-existent route or page, they are presented with an error message indicating that the requested content could not be found.
+- **Return to Home Button**: To enhance user navigation and convenience, a button is provided on the 404 Not Found Page. When clicked, this button redirects the user back to the home page.
 
-- Added feature whereby users are able to see how many spaces are available dynamically through the database.
+### Admin
 
-### Calendar
+- Describe the Admin page if applicable.
```
Comment (Charlie-robin) [inline README.md:192]: I think we can give a short overview of this page now.

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (luke-welbourn) [reply README.md]: added chu and jugz admin page notes

File: README.md (hunk 1 above)
Comment (luke-welbourn) [reply README.md]: changed to Union Client Project

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**